app/common/prescription/directives/prescription.js

```diff
@@ -679,6 +681,13 @@
       return angular.element('#addProviderAndPrescriptionDateModel');
     }
 
+    function verifyDrugAvailability(drug){
+      drugService.getDrugStock(drug,sessionService.getCurrentLocation()).then(function(data){
+        if(_.isEmpty(data.results)){
+         notifier.warning($filter('translate')('COMMON_MESSAGE_DRUG_WITHOUT_STOCK_AVAILABILITY'));;
+        }
+      });
```

Comment:
When this call fails the user should be notified that something went wrong. Should call ```.catch()``` here.

Reply:
there was a declared attribute in Batch resource that no longer existed in the domain class  that was causing this error.  Fixes by https://github.com/esaude/openmrs-module-inventory.poc/pull/7